gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -103,25 +107,43 @@ def main(args):
     models = {}
     applying_resources = {}
 
-    for region in GENOMIC_REGIONS:
-        for data_type in DATA_TYPES:
-            if (region == "autosome_par") | (data_type != "genomes"):
-                # Save a TableResource with a path to `preprocess_resources`
+    # For genomes need a preprocessed ht for autosome_par.
+    # For exomes need a preprocessed ht for autosome_par, chrX, and chrY.
+    # For context need a preprocessed ht for autosome_par, chrX, chrY, and
+    # full (autosome_par + chrX + chrY).
+    for data_type in DATA_TYPES:
+        if data_type != "genomes":
+            for region in GENOMIC_REGIONS:
                 preprocess_resources[(region, data_type)] = get_preprocessed_ht(
                     data_type, version, region, test
                 )
-        # Save a TableResource with a path to `training_resources`
-        training_resources[region] = get_training_dataset(version, region, test)
+        else:
+            preprocess_resources[(region, data_type)] = get_preprocessed_ht(
+                data_type, autosome_par, "autosome_par", test
+            )
 
-        for model_type in MODEL_TYPES:
-            # Save a path to `models`
-            models[(region, model_type)] = get_models(model_type, version, region, test)
+        if data_type == "context":
+            preprocess_resources[(region, data_type)] = get_preprocessed_ht(
+                data_type, version, "full", test
+            )
```

Comment:
I don't think this addition is working as expected, I made some changes that might be correct